content/news/public-oss-demo/2024/upcoming-live-update-november-2024.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Announcing PrestaShop Project Live Update November 2024"
+subtitle: "Join us on YouTube on November 27th, 2024 at 4pm (Paris time) for the next Live Update!"
+date:   2024-11-20
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+The next Live Update for the PrestaShop Project in 2024 will happen on **Wednesday, November 27th, 2024** at [4pm (Paris time)](https://time.is/1600_27_Nov_2024_in_Paris) on [YouTube](https://www.youtube.com/watch?v=9tLgjG8nYe4&ab_channel=PrestaShopProject)!
+
+We promised to bring you presentations from the PrestaShop Developer Conference 2024, and we are keeping our word!
+
+One of the highlights of this upcoming Live Update will be the presentation titled **"Long live Update Assistant!"** by members of the Seamless Upgrade & Extensibility team. This presentation, originally showcased at the PrestaShop Developer Conference 2024, has been highly anticipated by the community. It will be a great opportunity to learn more about the developments in the [autoupgrade](https://github.com/PrestaShop/autoupgrade) module, and to see how the *Update Assistant* will help you update your PrestaShop store to the latest version in the future.
```

Comment:
```suggestion
One of the highlights of this upcoming Live Update will be the presentation titled **"Long live the update assistant: strengthening trust in your shop's updates"** by members of the Seamless Upgrade & Extensibility team at PrestaShop SA. This presentation, originally showcased at the PrestaShop Developer Conference 2024, has been highly anticipated by the community. It will be a great opportunity to learn more about the developments in the [autoupgrade](https://github.com/PrestaShop/autoupgrade) module, and to see how the *Update Assistant* will help you update your PrestaShop store to the latest version in the future.
```